Aero390Spoilers/Ownship/Ownship.cs

```diff
@@ -203,39 +201,43 @@ public void RemoveEicasMessage(EICASMessage iMsg)
         public double GrossWeightLbs { get; set; }
         public double BaroSettingmmHg { get; set; }
         public double AltitudeASL { get; set; }
-        public int IasKts { get; set; }
+        public double IasKts { get; set; }
         public int IasOverspeed { get; set; }
         public int IasNES { get; set; }
         public int IasStall { get; set; }
         public double BankAngle { get; set; }
         public double AoA { get; set; }
         public double VS { get; set; }
+        public int InducedLift { get; set; }
         public string PhaseOfFlight { get; set; }
 
         //COCKPIT CONTROLS
         public int SpoilerLeverPosition { get; set; }
         public int FlapLeverPosition { get; set; }
-        public int SWControlWheelPosition { get; set; }
+        public double SWControlWheelPosition { get; set; }
         public int AutoBrakeSelectorPosition { get; set; }
-        public int LThrottlePosition { get; set; }
-        public int RThrottlePosition { get; set; }
+        public double LThrottlePosition { get; set; }
+        public double RThrottlePosition { get; set; }
 
         //DISPLAYS
         public int[] SpoilerDeflectionPercentage = new int[8];
+        public int MFS_Right = 0;
+        public int MFS_Left = 0;
+        public int MFS_as_brake = 0;
         public Stack<EICASMessage> EICASMessages = new Stack<EICASMessage>(); //Lines 1 to 8 for CW Messages, 9 to 11 for status
 
         //WARNING SYSTEM
         public bool WarningActive { get; set; }
         public bool CautionActive { get; set; }
 
         //MALFUNCTIONS AND SWITCHES
-        public bool Switch1On { get; set; }
+        public bool Switch1On { get; set; } = true;
```

Comment:
The initialization should not be done here. Please move it to the constructor.

Reply:
done!